app/models/user.rb

```diff
@@ -34,13 +34,10 @@ def friends
     friends_array = friendships.map { |friendship| 
       friendship.friend if friendship.confirmed
     }
-
-    friends_array + inverse_friendships.map { |friendship| 
+    friends_array = inverse_friendships.map { |friendship| 
       friendship.user if friendship.confirmed
     }
-
     friends_array.compact
-
   end
```

Comment:
# Great work @Qoosim 
Please merge and keep moving forward, I wish you the very best and see you at the very top 🚀

Reply:
Please consider that because of the scopes for friends and friend requests, the method above in the user model is now obsolete, as well as the method you had for the friend request